src/App.js

```diff
@@ -1,12 +1,18 @@
 import "./App.css";
 import { Routes, Route } from "react-router-dom";
 import Reviews from "./pages/Reviews";
+import Categories from "./pages/Categories";
+import Category from "./pages/Category";
+import NotFound from "./pages/NotFound";
 
 function App() {
   return (
     <div className="App">
       <Routes>
```

Comment:
I would suggest putting your reviews component on a path of `/` rather than `/reviews`